src/composables/copy.ts

```diff
@@ -0,0 +1,70 @@
+import { ref } from '@vue/composition-api';
+import { handleUnknownError } from '../popup/utils/helper';
+
+function fallbackCopyTextToClipboard(text: string) {
+  const textarea = document.createElement('textarea');
+  textarea.value = text;
+
+  // Avoid scrolling to bottom
+  textarea.style.top = '0';
+  textarea.style.left = '0';
+  textarea.style.position = 'fixed';
+
+  document.body.appendChild(textarea);
+  textarea.focus();
+
+  if (navigator.userAgent.match(/ipad|ipod|iphone/i)) {
+    textarea.contentEditable = 'true';
+    textarea.readOnly = true;
+
+    const range = document.createRange();
+
+    range.selectNodeContents(textarea);
+
+    const selection = window.getSelection();
+    if (selection) {
+      selection.removeAllRanges();
+      selection.addRange(range);
+    }
+    textarea.setSelectionRange(0, 999999);
+  } else {
+    textarea.select();
+  }
+
+  document.execCommand('copy');
+  document.body.removeChild(textarea);
+}
+
+export interface UseCopyOptions {
+  timeout: number
+}
+
+export function useCopy(options = {} as UseCopyOptions) {
```

Comment:
options: UseCopyOptions = {} but you cannot add empty options becouse in interface you have timeout number

Reply:
True, that was a mistake. Please see now.